src/cmd/linuxkit/run_hyperkit.go

```diff
@@ -42,7 +42,8 @@ func runHyperKit(args []string) {
 	mem := flags.Int("mem", 1024, "Amount of memory in MB")
 	var disks Disks
 	flags.Var(&disks, "disk", "Disk config. [file=]path[,size=1G]")
-	data := flags.String("data", "", "Metadata to pass to VM (either a path to a file or a string)")
+	data := flags.String("data", "", "String of metadata to pass to VM; overrides -data-path")
```

Comment:
I would error if a user specifies both instead of the override.

Reply:
Fine